agent/src/test/java/com/thoughtworks/go/agent/statusapi/AgentStatusHttpdTest.java

```diff
@@ -37,80 +50,169 @@ class AgentStatusHttpdTest {
     private AgentHealthHolder agentHealthHolder;
     @Mock
     private SystemEnvironment systemEnvironment;
-    @Mock
-    private NanoHTTPD.IHTTPSession session;
+
     private AgentStatusHttpd agentStatusHttpd;
 
+    private String hostName = "0.0.0.0";
+    private int port = 8093;
```

Comment:
I understand that when mocked correctly the server should not **actually** start, but can we change this to `127.0.0.1` or `localhost` so that it fails a bit more gracefully if the mocking needs changes in future (won't trigger firewall noise)?

Reply:
Can be done. Initially when writing the code, I have used loopback address, later I changed to the most generic. Let me change to loopback.

Reply:
Change committed.